configs/anyrtc.json

```diff
@@ -1,10 +1,81 @@
 {
   "index_name": "anyrtc",
   "start_urls": [
-    "https://docs.anyrtc.io/rtc-android/docs/"
+    {
+      "url": "https://docs.anyrtc.io/",
+      "selectors_key": "",
+      "tag": ["all"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtc-web/",
+      "selectors_key": "",
+      "tag": ["rtc-web"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtc-android/",
+      "selectors_key": "",
+      "tag": ["rtc-android"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtc-ios/",
+      "selectors_key": "",
+      "tag": ["rtc-ios"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtc-window/",
+      "selectors_key": "",
+      "tag": ["rtc-window"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtm-web/",
+      "selectors_key": "",
+      "tag": ["rtm-web"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtm-android/",
+      "selectors_key": "",
+      "tag": ["rtm-android"],
+      "variables": {
+        "lang": ["en", "zh-CN"]
+      }
+    },
+    {
+      "url": "https://docs.anyrtc.io/rtm-ios/",
+      "selectors_key": "",
+      "tag": ["rtm-ios"],
+      "variables": {
```

Comment:
This is not the correct usage of variables. Variables are used to be injected within the URL. [Please refer to the doc](https://docsearch.algolia.com/docs/config-file/#using-regular-expressions).